django_project/mezzanine_agenda/templates/agenda/event_detail.html

```diff
@@ -62,7 +62,7 @@ <h6 class="post-meta">
 {% endblock %}
 
 {% block event_detail_featured_image %}
-{% if settings.EVENT_USE_FEATURED_IMAGE and event.featured_image %}
```

Comment:
that why i'am not sure
that setting ("EVENT_USE_FEATURED_IMAGE") doesn't show on admin
and i think the default is false but we can't set as default due to not shown on admin

should i look on to it? 
but i think it is ok, because when user doesn't need image, he just deletes the featured image?